index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+<body>
+    <div class="main-header">
+        <div class="main-nav">
+            <div class="navigation-list"></div>
```

Comment:
Sry, but why do you not use `<ul>` instead of `<div>`

Reply:
Resolved.